hqscripts/management/commands/upload_to_s3.py

```diff
@@ -0,0 +1,56 @@
+from django.core.management.base import BaseCommand
+
+import boto3
+from botocore.exceptions import ClientError
+
+PROFILE_LENGTH = 4  # An entry for the profile name, then each of the 3 necessary keys
+
+
+class Command(BaseCommand):
+    help = 'Uploads a specified file to S3'
+
+    def add_arguments(self, parser):
+        parser.add_argument('filename')
+        parser.add_argument('bucket_name')
+        parser.add_argument('target_name')
+
+    def handle(self, *args, **options):
+        profile = self.get_profile()
+
+        if not self.is_valid_profile(profile):
+            self.stderr.write('Error with profile configuration')
+            exit(1)
+
+        self.upload_to_s3(options['filename'], options['bucket_name'], options['target_name'], profile)
+
+    def get_profile(self):
+        profile = {}
+
+        self.stdout.write('Paste S3 Profile: ')
+        for i in range(4):
+            try:
+                line = input('')
+            except EOFError:
+                break
+            if i > 0:
+                key, value, = [token.strip() for token in line.split('=', 1)]
+                profile[key] = value
+
+        return profile
+
+    def is_valid_profile(self, profile):
+        keys = ['aws_access_key_id', 'aws_secret_access_key', 'aws_session_token']
+        return not any(key for key in keys if key not in profile)
```

Comment:
The double negative is making this hard for me to understand. Would it be the same to say
```suggestion
        return all(key in profile for key in keys)
```

Reply:
Missed these suggestions. I assume `all` short-circuits when it finds a false value, so this seems like a good suggestion. I'll create a separate PR to fix this